Neos.Flow/Classes/Command/ResourceCommandController.php

```diff
@@ -232,13 +233,13 @@ public function cleanCommand()
         $assetRepository = class_exists(AssetRepository::class) ? $this->objectManager->get(AssetRepository::class) : null;
         /* @var ThumbnailRepository|null $thumbnailRepository */
         $thumbnailRepository = class_exists(ThumbnailRepository::class) ? $this->objectManager->get(ThumbnailRepository::class) : null;
-        $mediaPackagePresent = $assetRepository && $thumbnailRepository && $this->packageManager->isPackageAvailable('Neos.Media');
+        $mediaPackagePresent = $this->packageManager->isPackageAvailable('Neos.Media');
 
         if (count($brokenResources) > 0) {
             foreach ($brokenResources as $key => $resourceIdentifier) {
                 $resource = $this->resourceRepository->findByIdentifier($resourceIdentifier);
                 $brokenResources[$key] = $resource;
-                if ($mediaPackagePresent) {
+                if ($mediaPackagePresent && class_exists(AssetRepository::class) && class_exists(ThumbnailRepository::class)) {
```

Comment:
Yess ill be trying the null checks. Its not super straight forward to make phpstan work in this awkward situation.

The "original" intention is not older than a few weeky see https://github.com/neos/flow-development-collection/pull/3218 so that why i changed it so careless ;)

I probably now spend more time on trying phpstan to ignore this code than i would have to fix it properly 😂